tools/release/build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script defines the automated build step of the Datalab release process.
+#
+# That involves a clean build (not using the Docker image cache) of
+# the datalab-base, datalab-gateway, and datalab images; tagging the
+# resulting images with the current date; and then pushing the
+# datalab and datalab-gateway images to the Google Container Registry.
+
+PROJECT_ID="${PROJECT_ID:-cloud-datalab}"
+TIMESTAMP=$(date +%Y%m%d)
+LABEL="${LABEL_PREFIX:-}${TIMESTAMP}"
+BACKEND_IMAGE="gcr.io/${PROJECT_ID}/datalab-gateway:${LABEL}"
```

Comment:
I originally had these as just "GATEWAY_IMAGE" and "DATALAB_IMAGE", then changed them to backend/frontend to try to be more general.

The reasoning is that we've gotten some feedback that "Kernel Gateway" is jargon that won't be meaningful to anyone that isn't familiar with the internal workings of Jupyter.

As such, I was trying to avoid using that term unless necessary. That being said, I agree with the pushback that the main image isn't just a frontend.

I suppose it's also reasonable to expect anyone digging through the source code to be familiar with Jupyter terms, so I may switch back to "GATEWAY_IMAGE"/"DATALAB_IMAGE"